environment.rb

```diff
@@ -0,0 +1,84 @@
+module Environment
+  class Depot
+    attr_accessor :packs
+
+    def initialize
+      @packs = {
+        simple_transportation_pack: {
+          intelligence: [:cellphone],
+          item: %i[medipack chevy_vers]
+        },
+        standard_transportation_pack: {
+          intelligence: %i[cellphone antenna],
+          arsenal: [:colt_1911],
+          items: %i[handcuffs medipack chemistry chevy_versa]
+        },
+        simple_mission_pack: {
+          intelligence: %i[infopack laptop cellphone antenna],
+          arsenal: %i[colt_1911 handcuffs medipack chemistry financial]
+        },
+        standard_mission_pack: {
+          intelligence: %i[infopack laptop cellphone antenna],
+          arsenal: %i[remington_870 colt_1911 machete hatchet],
+          items: %i[handcuffs medipack chemistry financial]
+        }
+      }
+    end
+  end
+
+  class Control
+    attr_accessor :missions
+
+    @@mission_status = %w(active paused aborted accomplished failed)
+
```

Comment:
I would avoid this extra empty line here as `@@mission_status` is set and used right after.